app/components/wallet/AccountsOverview.js

```diff
@@ -195,6 +199,11 @@ class AccountsOverview extends Component<Props, State> {
     await getBalance();
   };
 
+  navigateToVault = () => {
```

Comment:
add TODO with comment that this is temporary solution so we won't forget to remove it

Reply:
fix